pages/components/slider.html

```diff
@@ -0,0 +1,213 @@
+            <li class="inline-block mx-1">
+              <a class="link" href="../getting-started/introduction.html">
+                Docs
+              </a>
+            </li>
+            <li class="inline-block mx-1">
+              <a
+                class="link"
+                href="https://github.com/hsnice16/PoshUI-Documentation"
+                target="_blank"
+              >
+                GitHub
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+
+      <!-- Header Docs Ends -->
+
+      <!-- Aside Left Starts -->
+
+      <aside class="aside-left">
+        <nav class="aside-container aside-left-container">
+          <ul class="mb-3">
+            <h3 class="my-1p5">Getting Started</h3>
+            <li>
+              <a href="../getting-started/introduction.html"> Introduction </a>
+            </li>
+            <li><a href="../getting-started/color.html">Color</a></li>
+            <li>
+              <a href="../getting-started/typography.html"> Typography </a>
+            </li>
+          </ul>
+
+          <ul class="mb-3">
+            <h3 class="my-1p5">Layout</h3>
+            <li><a href="../layout/css-grid.html">CSS Grid</a></li>
+          </ul>
+
+          <ul class="mb-3">
+            <h3 class="my-1p5">Components</h3>
+            <li><a href="alert.html">Alert</a></li>
+            <li><a href="avatar.html">Avatar</a></li>
+            <li><a href="badge.html">Badge</a></li>
+            <li><a href="button.html">Button</a></li>
+            <li><a href="card.html">Card</a></li>
+            <li><a href="image.html">Image</a></li>
+            <li><a href="input.html">Input</a></li>
+            <li><a href="list.html">List</a></li>
+            <li><a href="modal.html">Modal</a></li>
+            <li><a href="navigation.html">Navigation</a></li>
+            <li><a href="rating.html">Rating</a></li>
+            <li><a class="link-active" href="slider.html">Slider</a></li>
+            <li><a href="toast.html">Toast</a></li>
+            <li><a href="tooltip.html">Tooltip</a></li>
+          </ul>
+        </nav>
+      </aside>
+
+      <!-- Aside Left Ends -->
+
+      <!-- Main Docs Starts -->
+
+      <main class="main-docs">
+        <h1 class="mb-3">Slider</h1>
+        <p class="main-text">
+          Slider can be used to take user input based on some range.
+        </p>
+
+        <section class="my-2 section-component" id="input-slider">
+          <h2 class="my-2">Input Slider</h2>
+
+          <div class="component-container">
+            <div>
+              <input
+                type="range"
+                min="1"
+                max="5"
+                value="2"
+                step="0.1"
+                class="slider"
+              />
+            </div>
+
+            <pre class="pre-code">
+              <code class="language-html"> 
+&lt;input
+  type="range"
+  min="1"
+  max="5"
+  value="2"
+  step="0.1"
+  class="slider"
+/>
```

Comment:
suggestion - consider formatting your code

Reply:
Thanks for checking this, Prachi. Actually, the formatting we do inside `<code>` tag in HTML file gets applied on the preview as well. And I don't want any space on the preview that's why. It is like that.

Like, it looks like this now 
![image](https://user-images.githubusercontent.com/56081584/154902445-5347169c-77b4-44f2-bdcd-c5382390b1f8.png)

And if I will add space in the HTML file, then the space will be visible here too. At this place, 
![image](https://user-images.githubusercontent.com/56081584/154902562-9f9b2f29-7e0e-46b2-8d72-a990fb6ab939.png)
Which I don't want.